botocore/docs/method.py

```diff
@@ -209,6 +209,8 @@ def document_model_driven_method(section, method_name, operation_model,
         return_section.style.indent()
         return_section.style.new_line()
 
+        context = {'streaming_shape': operation_model.get_streaming_output()}
```

Comment:
Now I remembered that I checked that during my preliminary research, and I did a manual check just now to double confirm that, for the following sample model definition where the 2 members share the same `blobType` shape, they will end up as 2 different shape instances despite they are with the same type. So there would be no problem when we later use `if self._context.get('streaming_shape') == shape`.

            'shapes': {
                'OperationNameResponse': {
                    'type': 'structure',
                    'members': {
                        "Blob": {'shape': 'blobType'},
                        "Body": {'shape': 'blobType'}
                    },
                    'payload': 'Body'
                },
                'blobType': {
                    'type': 'blob'
                }
            }

So, no need to worry. It is a good question. Thanks!